src/components/views/auth/InteractiveAuthEntryComponents.tsx

```diff
@@ -782,7 +782,7 @@ export class RegistrationTokenAuthEntry extends React.Component<IAuthEntryProps,
 
 interface ISSOAuthEntryProps extends IAuthEntryProps {
     continueText?: string;
-    continueKind?: string;
+    continueKind?: Extract<AccessibleButtonKind, "primary" | "danger">;
```

Comment:
You asked for improving the Props of AccessibleButton which were untouched originally, the only way to do that was to make the types first match reality which had knock on effects.

Reply:
I don't remember asking for that, but even if I did: if, in the course of a review, you realise that we need a separate PR to lay some groundwork, that's a thing we should do rather than just scope-creep the PR ever larger.

Reply:
I mean, it's fine now you've done it, but really, this approach sucks.